indico/modules/events/util.py

```diff
@@ -60,7 +60,7 @@ def get_object_from_args(args=None):
         obj = event.contributions.filter_by(id=args['contrib_id'], is_deleted=False).first()
     elif object_type == 'subcontribution':
         obj = SubContribution.find(SubContribution.contribution.has(event_new=event, id=args['contrib_id'],
-                                                                    is_deleted=False)).first()
+                                                                    is_deleted=False), id=args['subcontrib_id']).first()
```

Comment:
Don't mix kwargs and starargs in ths same find() call - better use `SubContribution.id == ...` instead